ntp_operator.py

```diff
@@ -133,33 +141,74 @@ def _init_operator(self, idname: str, label: str) -> None:
         self._write("\tbl_options = {\'REGISTER\', \'UNDO\'}\n")
         self._write("\n")
 
-    def _is_outermost_node_group(self, level: int) -> bool:
```

Comment:
Now that we're no longer using inner functions for generated dependency node groups we can probably get rid of this `level` parameter we keep passing around everywhere, just include it as part of the write function